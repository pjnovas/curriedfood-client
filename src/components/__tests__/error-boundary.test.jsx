import ErrorBoundary from '../error-boundary';

takeSnapshots(
  {
    default: {
      children: 'Some component'
    }
  },
  ErrorBoundary
);
