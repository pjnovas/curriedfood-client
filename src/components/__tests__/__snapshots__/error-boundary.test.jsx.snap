// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default 1`] = `"Some component"`;
